elliottlib/cli/advisory_commons_cli.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import absolute_import, print_function, unicode_literals
+from elliottlib import logutil
+from elliottlib.cli.common import cli
+from elliottlib.util import green_prefix
+from errata_tool import Erratum, ErrataException
+import click
+
+LOGGER = logutil.getLogger(__name__)
+supported_update_fields = [
+    'publish_date',
+    'text_only',
+    'text_only_cpe',
+    'owner_email',
+    'manager_email',
+    'manager_id',
+    'qe_email',
+    'qe_group',
+]
+
+
+@cli.command("advisory-commons", short_help="Show or update advisory commons for a group of advisories")
+@click.option("--advisories", metavar='ADVISORY',
+              help="Select given advisory id(s). comma separated")
```

Comment:
Guess passing advisories as arguments in stead of options would be more ergonomics?
```suggestion
@click.argument("advisories", nargs=-1, type=click.IntRange(1), required=False)
```

Reply:
Yes! I didn't know argument could be required=False :) that is definitely better.